EPRN.Portal/Views/Waste/AccredidationLimit.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@model AccredidationLimitViewModel
+
+@{
+    ViewData["Title"] = @AccredidationLimitResources.Title;
+}
+
+<partial name="_ValidationSummary" />
+<partial name="_WasteTypeName" />
+
+<form method="post" action="Tonnes">
+    <input type="hidden" name="Id" value="@Model.JourneyId" />
```

Comment:
Don't need this - ID is provided through the routes binding

Reply:
The ID provided by route binding is consumed on the first pass-through the [Tonnes] action. 

On the second pass through (if needed) the ID is passed via hidden field in the view which is then "re-bound" to the route.

Reply:
I don't understand what you mean.